test/markdownlint-cli2-test-cases.js

```diff
@@ -444,6 +444,18 @@ const testCases =
     "usesRequire": true
   });
 
+  testCase({
+    "name": "markdownlint-cli2-config-option-with-extends",
+    "args": [
+      "--config",
+      "configs/custom.markdownlint-cli2.jsonc",
```

Comment:
This file could be named simply `.markdownlint-cli2.jsonc` which would be slightly clearer and get free schema highlighting.